### ISSUE TITLE
Home/End keys do 2 things in the command palette input

### ISSUE THREAD

**Issue by kamilmielnik** [metabase/metabase#42878]
### Describe the bug

https://github.com/metabase/metabase/assets/6830683/3a63d91a-f1b4-456a-a292-14124ad6a3e0

### To Reproduce

1. Hit <kbd>Ctrl</kbd> + <kbd>K</kbd> to open up the command palette
2. Type `hello` in the input
3. Hit <kbd>Home</kbd> key
    - :heavy_check_mark:  Cursor goes to the beginning of the input
    - :x: First result gets selected
4. Hit <kbd>End</kbd> key
    - :heavy_check_mark:  Cursor goes to the end of the input
    - :x: Last result gets selected

### Expected behavior

<kbd>Home</kbd> & <kbd>End</kbd> keys **do not affect** which result is selected.
<kbd>Home</kbd> & <kbd>End</kbd> keys **affect** cursor position in the input

### Information about your Metabase installation

master, 5d795be

### Severity

P3

**Comment by luizarakaki** [comment]
Idk, isn't the current behavior good? I think it will always do what you want.

**Comment by kamilmielnik** [comment]
@luizarakaki why did you close this? Pressing a button should not cause 2 unrelated things to happen.

https://github.com/user-attachments/assets/7d7d7376-34bd-4f15-b329-c13e9876a67e

**Comment by luizarakaki** [comment]
I don't think it is a problem. Picking one of these would be arbitrary too. I think this behavior works in most cases
